server/services/rpService.test.ts

```diff
@@ -272,6 +274,51 @@ describe('RpService', () => {
     expect(spy).toHaveBeenCalledWith('/resettlement-passport/prisoner/123/need/124', supportNeedsPatch)
   })
 
+  it('test getCaseNotesHistory returns case notes', async () => {
```

Comment:
(Unrelated to code change - just added to increase code coverage and make the pipeline happy)